samcli/local/events/api_event.py

```diff
@@ -82,6 +82,7 @@ def __init__(
         domain_name=None,
         request_time_epoch=int(time()),
         request_time=datetime.utcnow().strftime("%d/%b/%Y:%H:%M:%S +0000"),
+        operation_name=None,
```

Comment:
Is this used for all API Events? If so, shouldn't we model different RequestsContext's for different events?